examples/scripts/chat.py

```diff
@@ -220,7 +220,7 @@ def load_model_and_tokenizer(args):
         trust_remote_code=args.trust_remote_code,
         attn_implementation=args.attn_implementation,
         torch_dtype=torch_dtype,
-        device_map=get_kbit_device_map() if quantization_config is not None else None,
```

Comment:
I don't think the k-bit device map is actually needed for inference (one can shard and quantize AFAIK)